src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -0,0 +1,328 @@
+                    help="Physical interface(Multiples of 4)"
+		    ptype="UINT"
+		    >
+                </PARAM>
+            </PARAM>
+	    <PARAM
+	        name="Port-channel"
+		help="Port-channel ID (1-128)"
+		ptype="SUBCOMMAND"
+		mode="subcommand"
+		>
+		<PARAM
+		    name="po-id"
+                    help="port-channel id (1-128)"
+		    ptype="UINT"
+		    >
+                </PARAM>
+	    </PARAM>
+            </PARAM>
+        <ACTION>
+		if test "${if-subcommands}" = "Ethernet"; then&#xA;
+		    python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 interface Ethernet${phy-if-id}&#xA; 
+		elif test "${if-subcommands}" = "Port-channel"; then&#xA;
+		    python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 interface Port-channel${po-id}&#xA; 
+                else&#xA;
+		    python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 ${__full_line}&#xA;
+	        fi	
+        </ACTION>
+    </COMMAND>
+
+<!--====================show mac address-table static commands ======================-->
+    <COMMAND
+        help="mac address-table for static commands"
+        name="show mac address-table static">
+	<PARAM
+	    name="s-subcommands"
+	    help="static subcommands"
+	    mode="switch"
+	    ptype="SUBCOMMAND"
+	    optional="true"
+	    >
+	    <PARAM
+	        name="Address"
+		help="Address of static mac"
+		ptype="SUBCOMMAND"
+		mode="subcommand"
+		>
+		<PARAM
+		    name="mac-addr"
+                    help="Static Mac address"
+		    ptype="MAC_ADDR"
+		    >
+                </PARAM>
+            </PARAM>
+	    <PARAM
+	        name="VLAN"
+		help="VLAN ID (1-128)"
+		ptype="SUBCOMMAND"
+		mode="subcommand"
+		>
+		<PARAM
+		    name="vlan-id"
+                    help="static vlan id"
+		    ptype="VLAN_ID"
+		    >
+                </PARAM>
+            </PARAM>
+            <PARAM
+                name="Interface"
+                help="Static Interface"
+		ptype="SUBCOMMAND"
+		mode="subcommand"
+		>
+		<PARAM
+		    name="interface-subcommand"
+	            help="static interface subcommands"
+	            ptype="SUBCOMMAND"
+	            mode="switch"	    
+		    >
+                    <PARAM
+                        name="Ethernet"
+                        help="static interface Ethernet id - ethernet"
+		        ptype="SUBCOMMAND"
+		        mode="subcommand"
+		        >
+		        <PARAM
+		            name="phy-if-id"
+                            help="Static interface ethernet id-phy-if-id"
+			    ptype="UINT"
+			    >
+	                 </PARAM>
+		    </PARAM>
+                    <PARAM
+                        name="Port-channel"
+                        help="static port-channel id - pc"
+		        ptype="SUBCOMMAND"
+		        mode="subcommand"
+		        >
+		        <PARAM
+		            name="po-id"
+                            help="Static interface port-channel id po-id"
+			    ptype="UINT"
```

Comment:
Can we change PTYPE as LAG_ID? and in line number 303.

Reply:
Added LAG_ID in sonic-type and updated ptype for port-channel as LAG_ID.